word_list.py

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env python
 
-word_list = [" fuck ", " bitch ", " shit ", " tits ", " asshole ", " arsehole ", " cocksucker", " cunt", " hell ", " douche", " testicle", " twat", " bastard", " sperm", " shit", " dildo", " wanker", " prick", " penis", " vagina", " whore", " boner"]
\ No newline at end of file
+word_list = [" fuck ", " bitch ", " shit ", " tits ", " asshole ", " arsehole ", " cocksucker", " cunt", " hell ", " douche", " testicle", " twat", " bastard", " sperm", " shit", " dildo", " wanker", " prick", " penis", " vagina", " whore", " boner", "wank", "wanker", "fag", "faggot" ]
```

Comment:
Those words are substrings to match for. Writing "fag" as `"fag"`, for example, will also match `faggot`, `fagioli`, and `turfage`. `"wank"` likewise matches `wankada`, `wanker`, `swankiness`. You can control it by adding spaces to the front to limit it to a strict prefix, like what is done with some other words.

Reply:
Thanks for the tip, will add them shortly